### DISCUSSION TITLE
How to upload TCs

### DISCUSSION THREAD

**Discussion by junukwon7** [hydro-dev/Hydro#42] [Q&A] [answered]
I found out that the file upload button disappeared. I reinstalled hydro but it still has the same symptom.
Maybe changed at https://github.com/hydro-dev/Hydro/commit/e7794090b28c85b3978a0cd9aa18cac55b42b81d?

How do I upload TCs now?

![image](https://user-images.githubusercontent.com/48399106/104454360-126ff480-55e9-11eb-8753-94a679ee89c5.png)

**Comment by undefined-moe** [comment, marked as answer]
Click the `File` icon
![image](https://user-images.githubusercontent.com/29992205/104464241-a93eb000-55ed-11eb-995d-390382b80db2.png)
click `Add File` (blue button)
![image](https://user-images.githubusercontent.com/29992205/104464334-c5425180-55ed-11eb-976c-2a4ea7174783.png)

hint: select multiple files(unzipped), not a single zip file now.  
you can select and download/remove some files you nolonger needed.(download function seems doesn't work on some browsers)

the left side is testdata, and the right side is additional_files. All user with PERM_VIEW_PROBLEM can download additional_files (usually used to put some big samples) and you can use `[some description](file://filename of additional_file)` to point to the file in problem description.

by the way, the translation seems to be incorrect. can you figure out what causes the problem? (or what's the version you are using?)

**Reply by junukwon7** [reply]
Oh I got it. Thanks.
The version I was using was somehow broken because I modified it locally. I think that's why the translations were broken.

![image](https://user-images.githubusercontent.com/48399106/104528245-11bb7a80-564a-11eb-902e-42295730f4b7.png)
